lua/treesitter-context/context.lua

```diff
@@ -182,6 +169,29 @@ end
 
 local M = {}
 
+---@param bufnr integer
+---@param row integer
+---@param col integer
+---@return LanguageTree[]
+local function get_parent_langtrees(bufnr, range)
+  local root_tree = vim.treesitter.get_parser(bufnr)
+  if not root_tree then
+    return {}
+  end
+
+  local parent_langtrees = {root_tree}
+
+  while true do
+    child_langtree = parent_langtrees[#parent_langtrees]:language_for_range(range)
```

Comment:
```suggestion
    local child_langtree = parent_langtrees[#parent_langtrees]:language_for_range(range)
```

I can't see this declared anywhere.

Reply:
Done, I've also addressed a few linter errors that I overlooked.